bindings/muen/time.c

```diff
@@ -33,5 +35,25 @@ uint64_t solo5_clock_monotonic(void)
 /* return wall time in nsecs */
 uint64_t solo5_clock_wall(void)
 {
-    return tscclock_monotonic() + tscclock_epochoffset();
+  const struct muen_resource_type *const
```

Comment:
I think indentation is not correct. Should be 4 spaces AFAIK.

Reply:
Yes, I will do a lint at the end of this PR.